extension/content/loader.mjs

```diff
@@ -6,7 +6,7 @@
 
 const ONE_DAY = 1000 * 60 * 24;
 const LOCALES_URL =
-  "https://hg.mozilla.org/mozilla-central/raw-file/tip/browser/locales/all-locales";
+  "https://raw.githubusercontent.com/mozilla/gecko-dev/refs/heads/master/browser/locales/all-locales";
```

Comment:
This significantly improved the loading time for me (fetching the `hg.mozilla.org` url takes up to 17 _seconds_) but I'm not sure it's a good idea.

Reply:
We could definitely cache the locales but to you see an issue with pinging Github in particular? I thought mozilla-central will move to Github soon anyway and their infrastructure is just so much faster than ours.